packages/components/MenuButton/macos/MSFMenuButtonManager.m

```diff
@@ -0,0 +1,74 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+
+@implementation NSDictionary (MSFMenuButtonAdditions)
+
+- (BOOL)containsKey: (NSString *)key {
+     BOOL retVal = 0;
+     NSArray *allKeys = [self allKeys];
+     retVal = [allKeys containsObject:key];
+     return retVal;
+}
+
+@end
+
+@implementation RCTConvert (MSFMenuButtonAdditions)
+
++ (NSMenuItem *)menuItem:(id)json
+{
+  NSMenuItem *menuItem = [[NSMenuItem alloc] init];
+  [menuItem setTitle:[RCTConvert NSString:json[@"text"]]];
+  [menuItem setEnabled:![RCTConvert BOOL:json[@"disabled"]]];
+  [menuItem setToolTip:[RCTConvert NSString:json[@"title"]]];
+  [menuItem setIdentifier:[RCTConvert NSString:json[@"itemKey"]]];
+  return menuItem;
+}
+
++ (NSMenu *)NSMenu:(id)json
+{
+  NSMenu *menu = [[NSMenu alloc] init];
+  [menu setAutoenablesItems:NO];
+
+  NSArray *menuItems = [RCTConvert NSArray:json];
+  for (NSDictionary *menuItemJson in menuItems) {
+    NSMenuItem *menuItem = [RCTConvert menuItem:menuItemJson];
+
+    if ([menuItemJson containsKey:@"hasSubmenu"])
+    {
+      BOOL hasSubmenu = [RCTConvert BOOL:menuItemJson[@"hasSubmenu"]];
+      if (hasSubmenu) {
```

Comment:
How about using `objectForKey` instead of implementing an NSDictionary extension?

If `[RCTConvert BOOL:nil]` is OK and returns either `NO` or `nil`:
`if ([RCTConvert BOOL:[menuItemJson objectForKey:@"hasSubmenu"]]) {`

Otherwise add Elvis:
`if ([RCTConvert BOOL:[menuItemJson objectForKey:@"hasSubmenu"] ?: whateverThingRCTConvertIsExpectingThatReturnsNO]) {`

Reply:
(Blame my finely honed double-lookup senses from Python)

Reply:
`if ([menuItemJson objectForKey:@"hasSubmenu"])` works as expected. Thanks!